### PR TITLE
Related Posts: add docblock to existing filter

### PR DESCRIPTION
**Pull request by Umangvaghela**
#### Changes proposed in this Pull Request:

* Comment is not write on "jetpack_related_posts_customize_options filter" in class.ralated-posts-customize.php file so I add comment. 

#### Testing instructions:

*

### CODE REVIEW

File: modules/related-posts/class.related-posts-customize.php [hunk 1]
```diff
@@ -156,6 +156,13 @@ function get_options( $wp_customize ) {
 			restore_previous_locale();
 		}
 
+		/**
+		 * Filter that is used to customize related posts option.
```
Comment (jeherve) [inline modules/related-posts/class.related-posts-customize.php]: Could you clarify things a bit here? The filter allows you to change the options used to display Related Posts in the Customizer. I think this explains things a bit more.

File: modules/related-posts/class.related-posts-customize.php [hunk 2]
```diff
@@ -156,6 +156,13 @@ function get_options( $wp_customize ) {
...
+		 *
+		 * @since 4.4.0
+		 *
```
Comment (jeherve) [inline modules/related-posts/class.related-posts-customize.php]: Could you add `@module related-posts` here?

File: modules/related-posts/class.related-posts-customize.php [hunk 3]
```diff
@@ -156,6 +156,13 @@ function get_options( $wp_customize ) {
...
+		 * @param mixed array post customize option.
```
Comment (jeherve) [inline modules/related-posts/class.related-posts-customize.php]: This should always be an array, so I would remove "mixed". I would also add the array variable, like so:
`@param array $options Array of options used to display Related Posts in the Customizer.`

**Comment by Umangvaghela** [comment]
@jeherve 

Hello Sir,I update changes which suggested you. So please check it and provide your feedback if i miss something.

Thanks

**Pull request merged**